pkg/bazel/analysis_v2.proto

```diff
@@ -0,0 +1,226 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
```

Comment:
I noticed the v2 on this file. Note that I have basically no knowledge about the bazel proto and its versioning but it made me think: If there is a v2 then there was probably a v1 and could possibly be a v3. Could we get ourselves into a situation where the version of bazel being used causes problems here? Either with super old versions of bazel or future new versions?

Reply:
this was a strange case, specific to aquery, where they changed the data representation entirely. There was a `aquery --noincompatible_proto_output_v2` flag but it was default-off in bazel 4 and removed in bazel 5.
I don't think this is a pattern we'll have to worry about often, no.